email_sender/cmd/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/skobelina/email_sender/internal/mails"
+	"github.com/skobelina/email_sender/internal/mails/templates"
+	"github.com/skobelina/email_sender/pkg/queue"
+)
+
+var (
+	rabbitMQURL = os.Getenv("RABBITMQ_URL")
```

Comment:
can we move it to main function or create module which work with env/config variables ?

Reply:
yes